### DISCUSSION TITLE
"Update available" notification does not recommend latest version

### DISCUSSION THREAD

**Discussion by thgross** [pimcore/pimcore#9561] [Q&A / Support] [answered]
For example, in one installation, the "New Version Available!"-dialog says this:
Your Version: v10.0.1
New Version: 10.0.2
The current version is 10.0.6 though.

Another installation:
Your Version: v6.6.9
New Version: 6.8.0
Even though the current 6.x-Version is 6.9.6.

What is the logic behind the update recommendation?

**Comment by brusch** [comment, marked as answer]
You're right, there was something wrong - should be ok again 😊
Thanks for reporting!
